### DISCUSSION TITLE
haperos 小米14通知变形

### DISCUSSION THREAD

**Discussion by Sev73n** [MonwF/customiuizer#407] [Q&A]
![Screenshot_2023-12-08-11-18-09-681_com.miui.home.jpg](https://github.com/MonwF/customiuizer/assets/16495885/401e948c-f036-4ddc-9dca-2ca5ed3b4d89)

除了米客没有装其他的模块
![Screenshot_2023-12-08-11-18-09-681_com.miui.home.jpg](https://github.com/MonwF/customiuizer/assets/16495885/c0218921-397f-4fb2-a0fd-ad034539766d)

**Comment by MonwF** [comment]
想不到米客里的哪个功能会和这里有关联。。看下有没开彩色通知卡片

**Reply by Sev73n** [reply]
> 想不到米客里的哪个功能会和这里有关联。。看下有没开彩色通知卡片

![Screenshot_2023-12-08-11-33-50-974_name.monwf.customiuizer.jpg](https://github.com/MonwF/customiuizer/assets/16495885/5aad60dc-fc87-43d5-ab7e-47a5a1577e8f)

没有开 其实我也很纳闷 我米客开的修改很少 而且 只有QQ邮箱的通知出了问题 如果不是米客那就是开发版系统自己的bug了

**Reply by MonwF** [reply]
把米客的系统界面作用域关掉，重启下系统界面就知道了
